src/esn.calendar.libs/app/services/calendar-attendee-service.spec.js

```diff
@@ -24,14 +24,15 @@ describe('the calendarAttendeeService', function() {
       $provide.value('attendeeService', attendeeService);
     });
 
-    angular.mock.inject(function(_$rootScope_, _CAL_ATTENDEE_OBJECT_TYPE_, _CAL_ICAL_) {
+    angular.mock.inject(function(_$rootScope_, _CAL_ATTENDEE_OBJECT_TYPE_, _CAL_ICAL_, _$q_) {
       $rootScope = _$rootScope_;
       CAL_ATTENDEE_OBJECT_TYPE = _CAL_ATTENDEE_OBJECT_TYPE_;
       CAL_ICAL = _CAL_ICAL_;
+      $q = _$q_;
     });
   });
 
-  beforeEach(angular.mock.inject(function(_calendarAttendeeService_) {
+  beforeEach(inject(function(_calendarAttendeeService_) {
```

Comment:
In some other places you added `angular.mock`, but in this you removed it intentionally?

Reply:
For some cases like this an error was thrown: `angular.mock.inject` is not a function. Using `inject` makes it work fine.

Reply:
Mh... weird. Why is it inconsistent like that? There are some other `beforeEach` blocks that you use `angular.mock.inject` without errors.

Reply:
This one `inject` was originally used as a wrapper function and returns a function, maybe that's why. Will try to refactor this soon. I'm gonna left this conversation unresolved to remind myself.

Reply:
Done :trollface: